client/src/components/upload.css

```diff
@@ -19,48 +19,62 @@
   }
   
   .input-field input {
-    width: 200px;
+    
     padding: 10px;
     border: 1px solid #ccc;
     border-radius: 10px;
+    width: 150px;
+    height: 30px;
+    
+
   }
 
   .input1 {
     position: absolute;
-    left: 300px;
+    left: 340px;
+    top: -21px;
+    
+  }
+  .input2 {
+    position: absolute;
```

Comment:
Not a blocker for this PR but you might find it easier to use less `position: absolute` and use more natural html to layout elements.

Typically, you use position absolute when nothing else works 😄 The reason for this is it gets harder and harder to add more features the more you use absolute positioning.

You can do a lot with semantic html and a little bit of flex